EventListener/TagSubscriber.php

```diff
@@ -63,12 +86,16 @@ public function onKernelResponse(FilterResponseEvent $event)
     {
         $request = $event->getRequest();
         $response = $event->getResponse();
-
-        $tags = array();
+        if ($request->isMethodSafe()) {
```

Comment:
@ddeboer what do we want to do about this? allow to invalidate tags by adding them to the tagHandler in an unsafe request, or throw an exception in this case and tell that to invalidate the user should explicitly call invalidate?